lib/cereal/postgres/postgres.go

```diff
@@ -29,10 +29,11 @@ const (
 	//
 	defaultMaxIdleConnections = 4
 
-	enqueueTaskQuery   = `SELECT cereal_enqueue_task($1, $2, $3, $4)`
-	dequeueTaskQuery   = `SELECT * FROM cereal_dequeue_task($1)`
-	completeTaskQuery  = `SELECT cereal_complete_task($1, $2, $3, $4)`
-	getTaskResultQuery = `SELECT task_name, parameters, status, error, result FROM cereal_task_results WHERE id = $1`
+	enqueueTaskQuery      = `SELECT cereal_enqueue_task($1, $2, $3, $4)`
+	dequeueTaskQuery      = `SELECT * FROM cereal_dequeue_task($1)`
+	completeTaskQuery     = `SELECT cereal_complete_task($1, $2, $3, $4)`
+	getTaskResultQuery    = `SELECT task_name, parameters, status, error, result FROM cereal_task_results WHERE id = $1`
+	deleteTaskResultQuery = `DELETE FROM cereal_task_results WHERE id = $1`
```

Comment:
[style] There's that weird habbit of always using backticks for SQL.... It's not getting us anything if there's neither newlines nor quotes involved. 🤷‍♂ Fine.